client/components/input/input.html

```diff
@@ -1,3 +1,95 @@
-<link rel="stylesheet" href="./input.css">
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+  <meta charset="UTF-8">
+  <title>Input</title>
+  <link rel="stylesheet" href="./input.css">
+  
 
-<input class="unique-input-class" type="text" />
+</head>
+<body>
```

Comment:
You actually could keep whole `html`, `head` and `body` tag out of this (see what I mean in other components in project). Typically we need this on application pages, but for example component we could skip it, to simplify example code. Modern browsers will understand this, normally.